### PR TITLE
Remove redundant array creation

### PR DESCRIPTION
**Pull request by vzhikserg**
### Changes

Remove redundant array creation in log statements to simplify the code.

### CODE REVIEW

File: stream/distributedlog/protocol/src/main/java/org/apache/distributedlog/exceptions/UnsupportedMetadataVersionException.java [hunk 1]
```diff
@@ -25,6 +25,6 @@
     private static final long serialVersionUID = 4980892659955478446L;
 
     public UnsupportedMetadataVersionException(String message) {
-        super(StatusCode.UNSUPPORTED_METADATA_VERSION, String.format(message));
+        super(StatusCode.UNSUPPORTED_METADATA_VERSION, message);
```
Comment (eolivelli) [inline stream/distributedlog/protocol/src/main/java/org/apache/distributedlog/exceptions/UnsupportedMetadataVersionException.java]: I think this is a behavior change

Comment (vzhikserg) [reply stream/distributedlog/protocol/src/main/java/org/apache/distributedlog/exceptions/UnsupportedMetadataVersionException.java]: thanks for the comment. The change was reverted, but I don't see how this formatting can help without any arguments

Comment (eolivelli) [reply stream/distributedlog/protocol/src/main/java/org/apache/distributedlog/exceptions/UnsupportedMetadataVersionException.java]: You are right, but it is better not to change things if not needed or out of the scope of the patch.
Sorry for so deep nitpicking :)

**Comment by vzhikserg** [comment]
rebuild java8

**Comment by vzhikserg** [comment]
run pr validation

**Comment by vzhikserg** [comment]
@eolivelli thanks! Do I need to investigate the results from CI or it is needed to rerun them?

**Comment by eolivelli** [comment]
Something is broken on CI, the presence of Maven 3.5.
We have to change things config.
I will fix it as soon as I can, @sijie do you have time to fix it?

**Comment by eolivelli** [comment]
@sijie I will send a PR soon for an upgrade of Maven to latest version

**Comment by eolivelli** [comment]
this is the fix for the upgrade of Maven #2135

**Comment by eolivelli** [comment]
rebuild java8

**Comment by vzhikserg** [comment]
@eolivelli thanks a lot for your help!

**Comment by eolivelli** [comment]
run pr validation

**Comment by eolivelli** [comment]
You have checkstyle violations:
```

2019-07-28T14:55:14.731 [INFO] --- maven-checkstyle-plugin:3.0.0:check (default-cli) @ bookkeeper-server ---
2019-07-28T14:55:36.573 [INFO] Starting audit...
[ERROR] /home/jenkins/jenkins-slave/workspace/bookkeeper_precommit_pullrequest_validation/bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestRackawareEnsemblePlacementPolicy.java:2134: Line is longer than 120 characters (found 123). [LineLength]
[ERROR] /home/jenkins/jenkins-slave/workspace/bookkeeper_precommit_pullrequest_validation/bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestRackawareEnsemblePlacementPolicy.java:2142: Line is longer than 120 characters (found 123). [LineLength]
```

**Comment by eolivelli** [comment]
run this command to test your code for checkstyle

> mvn checkstyle:check -Dstream

**Comment by vzhikserg** [comment]
@eolivelli thanks, I don't know how I missed it, because right now I am adding the checkstyle validation to some pulsar modules ;)

**Pull request merged**